oshi-core/src/main/java/oshi/software/os/unix/aix/AixOSProcess.java

```diff
@@ -349,17 +356,19 @@ static State getStateFromOutput(char stateValue) {
         State state;
         switch (stateValue) {
         case 'O':
+            state = INVALID;
+            break;
+        case 'A':
             state = RUNNING;
             break;
-        case 'S':
-            state = SLEEPING;
+        case 'I':
+            state = NEW;
```

Comment:
In other OS's, I've not separately mapped New.  I think this happens if `fork()` is in the middle of operation, so it's probably reasonable to toss this in with waiting.

Reply:
Or if you keep the I, call it Idle.  But I prefer Waiting :)